.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js:
+  - 6
+  - stable
+
+branches:
+  only:
+    - master
```

Comment:
This shouldn't be needed either. It's not the default but I don't think it's necessary.